authors/apps/articles/models.py

```diff
@@ -88,6 +88,10 @@ class Comment(models.Model):
     )
     article = models.ForeignKey(Article, related_name='comments', on_delete=models.CASCADE)
     author = models.ForeignKey(Profile, related_name='comments', on_delete=models.CASCADE)
+    # The beginning of the highlight
+    begin = models.IntegerField(default=0)
```

Comment:
it's possible to achieve your feature without having  variables e.g end and start as field names. We need to keep our models and their fields as sacred as possible. In your case, a highlighted_text can be the field name and the ranges such as end and start can be variables you use in your methods and algos. LMK if this does not make sense.